7_react/react1/week1/src/App.js

```diff
@@ -0,0 +1,53 @@
+
+import './App.css';
+
+function App() {
+  return (
+    <div className="react-app">
+  <ToDoList/> 
+    </div>
+  );
+}
+
+export default App;
+
+const todo = [
+  { activity: "Get out of bed", date: "Wed Sep 13 2017"},
+  { activity: "Brush teeth", date: "Thu Sep 14 2017"},
+  { activity: "Eat breakfast", date: "Fri Sep 15 2017"},
+  ];
+
+
+  function Todo(props) {
+    const date = props.date; 
+    const activity = props.activity;
+   return(
+    <div>
+    <span className="activity-color">{activity} </span> 
+    <span className="date-color">{date}</span>
+  </div>
+   )
+}
+    
+    
+    function ToDoList() {
+    return (
+    <div>
+    {todo.map((todo) => {
+    const todoDate = todo.date;
+    const todoActivity = todo.activity;
```

Comment:
Here you have two variables with the same name:
* `todo` the array of data
* `todo` the item being iterated in the `map` function

It can easily become confusing for a developer to read such code. Consider to give unique names to all variables to make it easier to distinguish them from each other. A common practice is to give array type variables a plural name like `todos`. This makes it clear that it is an array that can contain multiple items. Each item you can then refer to as `todo`